packages/cli/src/commands/generate/job/templates/job.ts.template

```diff
@@ -0,0 +1,7 @@
+import { RedwoodJob } from '@redwoodjs/jobs'
+
+export class ${name}Job extends RedwoodJob {
+  async perform() {
```

Comment:
I think it would be good if the template had an argument to the `perform` method, to show how to work with the (generic) types on `RedwoodJob`. Could just take something simple, like a string or a number and then log it. That'd also show how to use the logger (or people will just fall back to what they know, i.e. `console.log`)